xblock/test/test_mixins.py

```diff
@@ -260,38 +271,151 @@ def _assert_node_attributes(self, node, expected_attributes):
             else:
                 self.assertIsNotNone(node.get(key))
 
-    def test_add_xml_to_node(self):
-        """ Tests add_xml_to_node with various field defaults and runtime parameters """
-        block_type = self.TestXBlock
-        block = self._make_block(block_type)
-        node = etree.Element(block_type.etree_node_tag)
+    def _assert_node_elements(self, node, expected_elements, check_none_attr=True):
+        """
+        Checks XML node elements to match expected elements.
+        """
+        node_elements = list(node)
+        self.assertEqual(set([elem.tag for elem in node_elements]), set(expected_elements.keys()))
+        if check_none_attr:
+            # All elements on the node are expected to have a "none"="true" attribute.
+            for elem in node:
+                self.assertEqual(elem.get('none'), 'true')
+
+    def test_no_fields_set_add_xml_to_node(self):
+        """
+        Tests that no fields are set on a TestXBlock when initially made
+        and no fields are present in the XML (besides force-exported defaults).
+        """
+        node = etree.Element(self.test_xblock_tag)
 
-        # precondition check
-        for field_name in block.fields.keys():
-            self.assertFalse(block.fields[field_name].is_set_on(block))
+        # Precondition check: no fields are set.
+        for field_name in self.test_xblock.fields.keys():
+            self.assertFalse(self.test_xblock.fields[field_name].is_set_on(self.test_xblock))
 
-        block.add_xml_to_node(node)
+        self.test_xblock.add_xml_to_node(node)
 
         self._assert_node_attributes(
-            node, {'simple_force_export': 'default', 'unique_id_force_export': UNIQUE_ID}
+            node,
+            {
+                'str_str_default_force_export': 'default',
+                'str_uid_default_force_export': UNIQUE_ID
+            }
         )
+        self._assert_node_elements(
+            node,
+            {
+                # The tags are prefixed with {namespace}.
+                '{{{}}}{}'.format(XML_NAMESPACES["option"], tag): None
+                for tag in ['str_none_default_force_export']
+            }
+        )
+
+    def test_set_fields_add_xml_to_node(self):
+        """
+        Tests that set fields appear in XML after add_xml_to_node.
+        """
+        node = etree.Element(self.test_xblock_tag)
 
-        block.field = 'Value1'
-        block.simple = 'Value2'
-        block.simple_force_export = 'Value3'
-        block.unique_id = 'Value4'
-        block.unique_id_force_export = 'Value5'
+        self.test_xblock.str_field = 'Value1'
```

Comment:
Could you make these values more descriptive of what they are? Like `"string_field"` instead of `"Value1"`?